src/components/ItemUnsupported.tsx

```diff
@@ -0,0 +1,28 @@
+import { useTranslation } from 'react-i18next';
+import { StyleSheet, View } from 'react-native';
+import { Text } from 'react-native-elements';
+
+const ItemUnsupported = () => {
+  const { t } = useTranslation();
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.text}>
+        {t('This file cannot be opened in the mobile app')}
+      </Text>
```

Comment:
Maybe we can add a suggestion to contact us in order to request support for some file types ? This way we can reduce frustration and learn what we need to implement in priority.